homeassistant/components/toon.py

```diff
@@ -0,0 +1,146 @@
+"""
+Toon van Eneco Support.
+This provides a component for the rebranded Quby thermostat as provided by
+Eneco.
+"""
+import logging
+import voluptuous as vol
+
+# Import the device class from the component that you want to support
+from homeassistant.const import (CONF_USERNAME, CONF_PASSWORD)
+from homeassistant.helpers.discovery import load_platform
+import homeassistant.helpers.config_validation as cv
+
+# Home Assistant depends on 3rd party packages for API specific code.
+REQUIREMENTS = ['toonlib==1.0.0']
+
+_LOGGER = logging.getLogger(__name__)
+
+DOMAIN = 'toon'
+TOON_HANDLE = 'toon_handle'
+CONF_GAS = 'gas'
+DEFAULT_GAS = True
+CONF_SOLAR = 'solar'
+DEFAULT_SOLAR = False
+
+# Validation of the user's configuration
+CONFIG_SCHEMA = vol.Schema({
+    DOMAIN: vol.Schema({
+        vol.Required(CONF_USERNAME): cv.string,
+        vol.Required(CONF_PASSWORD): cv.string,
+        vol.Optional(CONF_GAS, default=DEFAULT_GAS): cv.boolean,
+        vol.Optional(CONF_SOLAR, default=DEFAULT_SOLAR): cv.boolean,
+    }),
+}, extra=vol.ALLOW_EXTRA)
+
+
+def setup(hass, config):
+    """Setup toon."""
+    if CONF_GAS in config['toon']:

```

Comment:
No need to check if it is in the config, you set a default value so that will always be there.